frontend/src/context/AuthContext.jsx

```diff
@@ -49,6 +49,11 @@ const value = {
     loading
 };
 
-return <AuthContext.Provider value={value}>{!loading && children}</AuthContext.Provider>;
+return (
+    <AuthContext.Provider value={value}>
+        {!loading && children}
+        </AuthContext.Provider>
```

Comment:
it seems this file needs to be formatted properly, @imanihenderson could you please take some time to setup the ESLint + Prettier to auto-format the code for you?
https://eslint.org/
https://prettier.io/

James had successfully set that up in his project, please feel free to reach out him for help :)